bedrock/mozorg/templates/mozorg/developer/index.html

```diff
@@ -0,0 +1,435 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "base-pebbles.html" %}
+
+{% block page_title %}{{ _('Developer Tools &amp; Resources') }}{% endblock %}
+{% block page_title_suffix %} | Mozilla{% endblock %}
+
+{% block page_desc %}{{ _('Find tools, resources, links videos and more for Mozilla developers.') }}{% endblock %}
+
+{% block page_image %}{{ static('img/mozorg/developer/hub/page-image.png') }}{% endblock %}
+
+{% block body_id %}developer{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'developer-hub' %}
+{% endblock %}
+
+{% block site_header %}
+  {% if LANG.startswith('en-') %}
+    {% include 'includes/global-nav.html' %}
+  {% else %}
+    {% set logo_src = static('img/pebbles/moz-wordmark-dark-reverse.svg') %}
+    {% include 'mozorg/home/includes/nav.html' %}
+  {% endif %}
+{% endblock %}
+
+{% macro lazy_image(image_name) -%}
+<div class="image-container">
+  <img src="{{ static('img/mozorg/developer/hub/placeholder.png') }}" data-src="{{ static('img/mozorg/developer/hub/' + image_name) }}" alt="">
```

Comment:
Don't blame me, it's your placeholder ;) Maybe just a really light gray, perhaps with a little "m" icon in the middle?